### DISCUSSION TITLE
issue: Trigger validation on computed input doesn't seem to work

### DISCUSSION THREAD

**Discussion by clanga-flywire** [react-hook-form/react-hook-form#9918] [Q&A] [answered]
### Version Number

7.43.1

### Codesandbox/Expo snack

https://codesandbox.io/s/blissful-cdn-37lbol?file=/src/App.js

### Steps to reproduce

1. Have a form with mode `onChange`.
2. Create two inputs that have the value computed based on the other.
3. Call trigger to validate form after setting the value on the other input.

5. Type a value in one filed until the validation fails.
6. See how the error is displayed only for the field that you typed even though the other should fail as well.

### Expected behaviour

Both fields to fail validation and display an error

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by bluebill1049** [comment, marked as answer]
`maxLength` is used for string value and you are sending `number` type.

**Reply by leapful** [reply]
@clanga-flywire 

Please set field value in correct type with the built-in validation to make it work.

Working demo: https://codesandbox.io/s/unruffled-tamas-t9vgdq?file=/src/App.js

**Reply by clanga-flywire** [reply]
It works perfectly. Thank you
